ZeroWasteFriendlyTests/ActivityTiles/FakeActivityTilesPresenter.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  FakeActivityTilesPresenter.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 05/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+@testable import ZeroWasteFriendly
+
+final class FakeActivityTilesPresenter: ActivityTilesPresenter {
+
+    var toogleSpinnerCalled: Bool?
+    var toogleSpinnerStateCalled: Bool?
+    var presentAlertCalled: Bool?
+    var presentAlertTitleCalled: String?
+    var presentAlertMessageCalled: String?
+    var presentErrorCalled: Bool?
+    var presentErrorErrorCalled: Error?
+    var presentActivityTilesCalled: Bool?
+
+    func toggleSpinner(_ state: Bool) {
+        toogleSpinnerCalled = true
+        toogleSpinnerStateCalled = state
+    }
+
+    func presentAlert(title: String, message: String) {
+        presentAlertCalled = true
+        presentAlertTitleCalled = title
+        presentAlertMessageCalled = message
+    }
+
+    func presentError(_ error: Error) {
+        presentErrorCalled = true
+        presentErrorErrorCalled = error
```

Comment:
you'll need to build more universal solution, you have spinner presenters and alert presenters so they're not copy-pasted in all controllers and presenters

if you do this, then you'll have these tests everywhere which is also bad, need to distinguish between shared presenter and controller code

will you be able to try it on our own or need my help?